src/main/java/seedu/address/logic/commands/edit/EditUserCommand.java

```diff
@@ -74,23 +78,28 @@ public CommandResult execute(Model model) throws CommandException {
         Person userToEdit = model.getUser();
 
         Person editedUser = createEditedUser(userToEdit, editUserDescriptor);
+
+        if (editedUser.getFrom().compareTo(editedUser.getTo()) > 0) {
+            throw new CommandException(MESSAGE_FROM_TO_MISORDER);
+        }
+
         model.setUser(editedUser);
         model.commitResumeBook();
 
         return new EditUserCommandResult(editedUser.toString(),
-                String.format(MESSAGE_EDIT_PERSON_SUCCESS, editedUser),
-                model.getDisplayType());
+                MESSAGE_EDIT_PERSON_SUCCESS, "");
```

Comment:
It's not safe to not pass up model.getDisplayType()

Reply:
Thanks bruh :)